### DISCUSSION TITLE
Should I disable MQTT gateway setting?

### DISCUSSION THREAD

**Discussion by gkathan** [zwave-js/zwave-js-ui#3616] [Q&A] [answered]
hi,
I have (probably a dumb) question:
* I am using zwave-js with 30 devices and just realized that in settings I have the mqtt gateway disabled
* still everything is working 
* is there any benefit by enabling the mqtt gateway option ?
happy for any advice
cheeerz from vienna![Screenshot_20240223-133654.png](https://github.com/zwave-js/zwave-js-ui/assets/6329814/8e0da83f-0e10-43e9-a0c4-f53ba1c6d080)

**Comment by robertsLando** [comment, marked as answer]
@gkathan The gateway functionality is only needed if you wish to use MQTT otherwise it's better to keep it disabled

**Comment by gkathan** [comment]
ok thänx very much for this clearance
cheeerz from.vienna

gerold kathan :: +43 699 1973536
